src/services/at-sce-api.service.js

```diff
@@ -39,9 +50,17 @@ AT_SCE_SERVICE.getAllUsers = () => {
   });
 };
 
-// Operation: Get USER by ID - GET/api/v1/users/{id}
-AT_SCE_SERVICE.getUserById = () => {
-  return {};
+// Operation: Get USERS by ID - GET/api/v1/users/{id}
+AT_SCE_SERVICE.getUsersById = (id) => {
+  return axios({
+    method: "GET",
+    url: `${AT_SCE_SERVICE_URI}/v1/users/` + id,
```

Comment:
you can use template literals here https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals